externals/paho-mqtt/v0/src/index.kt

```diff
@@ -0,0 +1,63 @@
+//headers for Paho JavaScript Client
+//The Paho JavaScript Client is an MQTT browser-based client library written in Javascript that uses WebSockets to connect to an MQTT Broker.
+//https://github.com/eclipse/paho.mqtt.javascript
+
+package js.externals.paho_mqtt
+
+external object Paho {
```

Comment:
Have you tried to convert `https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/paho-mqtt/` using [ts2kt](https://github.com/Kotlin/ts2kt/)?

Reply:
I tried ts2kt on paho-mqtt typescript but I got an error.
I tried it on jquery typescript and everything was fine... so I think my proceedings were all right.

This is the error I got 
ts2kt version: 0.1.3
Converting index.d.ts
fs.js:640
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^

Error: ENOENT: no such file or directory, open 'C:\module.d.ts'
    at Error (native)
    at Object.fs.openSync (fs.js:640:18)
    at Object.fs.readFileSync (fs.js:508:33)
    at getScriptSnapshotFromFile (C:\Users\Simone\AppData\Roaming\npm\node_modules\ts2kt\ts2kt.js:5398:46)
    at translate (C:\Users\Simone\AppData\Roaming\npm\node_modules\ts2kt\ts2kt.js:5623:21)
    at translateToDir (C:\Users\Simone\AppData\Roaming\npm\node_modules\ts2kt\ts2kt.js:5177:26)
    at main (C:\Users\Simone\AppData\Roaming\npm\node_modules\ts2kt\ts2kt.js:5387:5)
    at C:\Users\Simone\AppData\Roaming\npm\node_modules\ts2kt\ts2kt.js:6243:3
    at Object.<anonymous> (C:\Users\Simone\AppData\Roaming\npm\node_modules\ts2kt\ts2kt.js:6246:2)
    at Module._compile (module.js:570:32)

Reply:
Could you please [file an issue to Kotlin/ts2kt](https://github.com/Kotlin/ts2kt/issues/new)